### PR TITLE
add sentry with npx wizard

### PR DESCRIPTION
**Pull request by b-pmcg**
### What does this PR do?
adds sentry sdk and dsn to track application and API errors

### Steps for testing:
see the sentry dashboard filling up with new errors

### Screenshots (if relevant):
Screen Shot 2021-09-20 at 3 30 49 PM

### Any additional helpful information?:
The DSN can be kept public, see: https://github.com/getsentry/sentry-docs/pull/1723

### CODE REVIEW

**Comment by kaylanavelle** [comment]
Nice, looks good.  We can adjust settings as we go.  So the CircleCI config changes mean we'll get sentry notifications if there's errors in our automated tests?

**Review by kaylanavelle — APPROVED** [review]

**Review by tyler17 — APPROVED** [review]
Nice, looks good.  We can adjust settings as we go.  So the circleCI config changes mean we'll get sentry notifications for the automated tests?

**Review by adamgoth — APPROVED** [review]

File: sentry.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=makerdao-dux
+defaults.project=makerdao-dux
+cli.executable=../../../../.npm/_npx/59294/lib/node_modules/@sentry/wizard/node_modules/@sentry/cli/bin/sentry-cli
```
Comment (rafinskipg) [inline sentry.properties:4]: this cli executable path looks weird

**Pull request merged**